src/client/src/ui/analytics/views/analyticsView.jsx

```diff
@@ -1,26 +1,26 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import classnames from 'classnames';
-import { router, logger } from '../../../system';
-import { ViewBase } from '../../common';
-import { AnalyticsModel, PositionsChartModel, PnlChartModel } from '../model';
+import { logger } from '../../../system';
+import { PositionsChartModel, PnlChartModel } from '../model';
 import { ChartGradient } from './';
 import NVD3Chart from 'react-nvd3';
 import AnalyticsBarChart from './chart/analyticsBarChart';
 import numeral from 'numeral';
-import Dimensions from 'react-dimensions';
 import PositionsBubbleChart from './positions-chart/positionsBubbleChart';
 import './analytics.scss';
 
 var _log:logger.Logger = logger.create('AnalyticsView');
 
-@Dimensions()
-export default class AnalyticsView extends ViewBase {
+export default class AnalyticsView extends React.Component {
+
+  static propTypes = {
+    model: React.PropTypes.object.isRequired,
```

Comment:
There is some circular dependency issues with the require statement whereby if you require the model here for the type of check it sometimes works and sometimes is undefined (the model also requires the view for view  resolution).  I didn't have time to look into it further.